rpcs3/Emu/RSX/VK/VKTexture.cpp

```diff
@@ -472,6 +472,7 @@ namespace vk
 					stretch_image_typeless_safe(src, dst, typeless_stencil->value, src_rect, dst_rect, depth_stencil, VK_IMAGE_ASPECT_STENCIL_BIT);
 					break;
 				}
+				default: break;
```

Comment:
fmt::throw("Unreachable" HERE);
If this happens, it will be unrecoverable driver crash.